doc/conf.py

```diff
@@ -110,10 +110,11 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-#html_title = None
+# Don't show version in title
+html_title = "FOSHttpCache Documentation"
 
 # A shorter title for the navigation bar.  Default is the same as html_title.
-#html_short_title = None
+html_short_title = FOSHttpCache
```

Comment:
seems the example was misleading. this needs quotes i think. travis doc building fails.